packages/core/test/lib/console-child.js

```diff
@@ -0,0 +1,86 @@
+const assert = require("chai").assert;
+const fs = require("fs-extra");
+const path = require("path");
+const tmp = require("tmp");
+const TruffleConfig = require("@truffle/config");
+const { deriveConfigEnvironment } = require("../../lib/command-utils");
+
+let config;
+
+function createSandbox(source) {
+  if (!fs.existsSync(source)) {
+    throw new Error(`Sandbox failed: source: ${source} does not exist`);
+  }
+
+  const tempDir = tmp.dirSync({ unsafeCleanup: true });
+  fs.copySync(source, tempDir.name);
+  const config = TruffleConfig.load(
+    path.join(tempDir.name, "truffle-config.js"),
+    {}
+  );
+  return config;
+}
+
+describe("console-child", function () {
+  before("Create a sandbox", function () {
+    config = createSandbox(
+      path.join(__dirname, "..", "sources", "console-child")
+    );
+  });
+
+  describe("deriveConfigEnvironment", function () {
+    it("network has provider", function () {
```

Comment:
These strings should read as plain english descriptions of what these things do. An example is something like
```
it("returns a config with...<whatever you expect>...
```

Reply:
I will change the strings according to your suggestions @eggplantzzz. Thanks !